### PR TITLE
(DPS-3269) New docs sidebar - MVP

### PR DESCRIPTION
**Pull request by arturfracala**
# 🚀 Links

- [Jira](https://livechatinc.atlassian.net/browse/DPS-3269)

# 📓 Description

### CODE REVIEW

File: src/components/core/SideNav/SideNav.js [hunk 1]
```diff
@@ -1,138 +1,131 @@
...
+  transform: "rotate(270deg)",
+  transition: "transform 300ms",
+};
+const closeIconStyle = {
+  transform: "rotate(90deg)",
+  transition: "transform 300ms",
+};
 
-  const navColor = getVersionColor(selectedVersion, versions);
+const SideNav = ({ category, version = "3.3", title }) => {
```
Comment (jakubsikora) [inline src/components/core/SideNav/SideNav.js:75]: Here again with the version 3.3. Same as above.

File: src/components/core/components.js [hunk 2]
```diff
@@ -132,34 +133,29 @@ export const Content = styled.article`
...
-  max-width: 100%;
+  width: 100%;
   padding-bottom: ${({ noPadding }) => (noPadding ? "0" : "30vh")};
   position: relative;
-  margin-left: ${({ noMargin }) => (noMargin ? "0" : "50px")};
-  width: 100%;
+  @media (min-width: 768px) {
+    margin-left: ${({ noMargin }) => (noMargin ? "0px" : "260px")};
+  }
+  width: ${({ fullWidth }) => (fullWidth ? "calc(100%-260px)" : "900px")};
```
Comment (jakubsikora) [inline src/components/core/components.js:158]: So you have width 100%, and then this ?

File: src/pages/_app.js [hunk 3]
```diff
@@ -20,7 +19,7 @@ function MyApp({ Component, pageProps }) {
     require("smooth-scroll")('a[href*="#"]:not([href="#open-chat"])', {
       speed: 160,
       speedAsDuration: true,
-      offset: SCROLL_OFFSET,
+      offset: 130,
```
Comment (jakubsikora) [inline src/pages/_app.js:22]: Can we use `SCROLL_OFFSET` again ?

**Review by jakubsikora — CHANGES_REQUESTED** [review]
Nice work! Few comments.

File: src/components/core/SideNav/NestedMenu.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useRouter } from "next/router";
+import { string } from "prop-types";
+import containers from "../../../configs/containers.json";
+import articles from "../../../configs/articles.json";
+import MenuItem from "./MenuItem";
+
+const NestedMenu = ({ category, version = "3.3" }) => {
```
[the remark this answers is not in the document]
Comment (arturfracala) [reply src/components/core/SideNav/NestedMenu.js]: Good point, now I use const values from `constant` directory

File: src/components/core/components.js (hunk 2 above)
Comment (arturfracala) [reply src/components/core/components.js:157]: Yeah, you're right

File: src/pages/_app.js (hunk 3 above)
Comment (arturfracala) [reply src/pages/_app.js]: Yes, we can 😅 already done it

**Review by jakubsikora — APPROVED** [review]

**Pull request merged**